Tile.cpp

```diff
@@ -61,89 +61,7 @@ worn_item::worn_item()
     dyematt.type = -1;
 }
 
-Tile::Tile()
-{
-    // zero out everything
-    valid = false;
-    visible = false;
-
-    ownerSegment = nullptr;
-
-    x = 0;
-    y = 0;
-    z = 0;
-    tileType = tiletype::Void;
-    material.type = 0;
-    material.index = 0;
-    layerMaterial.type = 0;
-    layerMaterial.index = 0;
-    veinMaterial.type = 0;
-    veinMaterial.index = 0;
-    hasVein = false;
-
-    depthBorderNorth = false;
-    depthBorderWest = false;
-    depthBorderDown = false;
-
-    shadow = 0;
-
-    wallborders = 0;
-    floorborders = 0;
-    openborders = 0;
-    rampborders = 0;
-    upstairborders = 0;
-    downstairborders = 0;
-    lightborders = 0;
-
-    fog_of_war = false;
-
-    rampindex = 0;
-
-    deepwater = 0;
-
-    flow_direction = tile_liquid_flow_dir::none;
```

Comment:
This isn't necessarily the same as memset(0) on the field, did you check?

Reply:
`tile_liquid_flow_dir::none` is zero, so this one's fine.